litestar/utils/helpers.py

```diff
@@ -102,3 +103,25 @@ def get_exception_group() -> type[BaseException]:
         from exceptiongroup import ExceptionGroup as _ExceptionGroup  # pyright: ignore
 
         return cast("type[BaseException]", _ExceptionGroup)
+
+
+def envflag(varname: str, default: bool = False) -> bool:
```

Comment:
This does not behave like click. It will consider all values it doesn't consider `True` to be `False`, so e.g. a typo like `Tru` or `2` would also be considered `False`.

It should raise in that case instead. Please consult the click docs in the original issue I've linked on the actual behaviour